front/src/components/atoms/Button.tsx

```diff
@@ -19,4 +22,23 @@ export const Button = ({ children, onClick, className, colour = "blue", ...props
             {children}
         </button>
     );
-}
\ No newline at end of file
+};
+
+const colourPicker = (colour: string): string => {
+    const blueStyle = "bg-theodo-turquoise hover:bg-omnilog-clear-blue";
```

Comment:
Is this colourPicker specific to the button? If so, maybe include that in the name? Before I understood the function I thought if might be better living outside this folder, but actually it's looks specific to button? Probably good to make this clear in the name